lib/csv.rb

```diff
@@ -1427,13 +1477,12 @@ def init_separators(col_sep, row_sep, quote_char, force_quotes)
     # store the selected separators
     @col_sep    = col_sep.to_s.encode(@encoding)
     if @col_sep == " "
-      @col_sep_split_separator = Regexp.new(/#{Regexp.escape(@col_sep)}/)
+      @col_sep_split_separator = "\s"
```

Comment:
Can we use `= @col_sep` instead of `= "\s"` here?

Reply:
Exactly, we can use @col_sep.
@col_sep_split_separator unified to @col_sep.
e044bc9